### PR TITLE
Fix Issue #64- Add Privacy Policy

### LINKED ISSUE #64: Add Privacy Policy Page
We need to have the privacy policy on our website.

### PR DESCRIPTION
**Pull request by FrancescoSTL**
Hey @purukaushik, I added the Privacy Policy page. I added a privacy policy page along with changing around the menu to have download at the end on the right and bolding it. I'm wondering if we should make download stick to the right hand side of the page and be a different color than the other menu items. Also, we probably either need to dynamically populate the href to that based upon the user agent of the user.

Let me know what you think about styling, etc.

r?

Fixes #64

### CODE REVIEW

**Comment by FrancescoSTL** [comment]
Looks like I might've messed up this css. Fixing.
screen shot 2016-08-29 at 11 54 49 am

**Comment by FrancescoSTL** [comment]
Ok, fixed. @purukaushik r?

File: views/index.html [hunk 1]
```diff
@@ -80,8 +80,9 @@
         <ul class="nav navbar-nav">
           <li class="active"><a href="/">Home</a></li>
           <li><a href="/dashboard">Dashboard</a></li>
-          <li><a href="https://github.com/FrancescoSTL/Site-Sonar">Download</a></li>
+          <li><a href="/privacy">Privacy</a></li>
```
Comment (purukaushik) [inline views/index.html:83]: Add this here too? - [L#165](https://github.com/FrancescoSTL/Site-Sonar-Dashboard/blob/7e1ca358181cf3dad0c80f002a8ef28b82d09816/views/index.html#L165)
i.e privacy policy at bottom of page.

Comment (FrancescoSTL) [reply views/index.html:83]: @purukaushik Ah, yep, I missed that. I'll add it to the footers.

**Comment by FrancescoSTL** [comment]
Done. @purukaushik r?

**Comment by purukaushik** [comment]
@FrancescoSTL  add footer to index.html as well. I dunno why we  have to do this in multiple places. I guess we can refactor this once we have a UX/UI guy on board after v1.0

**Comment by purukaushik** [comment]
BTW you can view your changes on the dev site. Once this is merged I will rebase my changes in #68 and deploy in dev so you can review it directly on the dev site too.

**Comment by FrancescoSTL** [comment]
Fixed. Merging.

**Pull request merged**